README.md

```diff
@@ -320,5 +201,120 @@ This request will delete webhook permanently
 #### Refresh webhook
 If the number of failed requests wil exceed `WEBHOOK_MAXTRIES`, webhook will be set to inactive. To refresh webhook you can use this same endpoint as for webhook creation.
 
+### Running from source
+
+1. Install Go according to the installation instructions here: http://golang.org/doc/install
+
+Options to run Pulse:
+
+a) Clone the repo
+
+   ```sh
+  git clone https://github.com/bitcoin-sv/pulse
+   ``` 
+1. ```go run ./cmd/main.go```
+
+Or run app with docker
+1. ```docker compose up --build```
+
+b) Get package from ``pkg.dev.go``
+1. ```go get -u https://pkg.go.dev/github.com/bitcoin-sv/pulse```
+2. ```go build -o pulse```
+3. ```./pulse```
+
+
+### Config Variables
+
+Default config variables can be overridden by (in this order of importance):
+
+1. Flags (only the ones below)
+2. ENV variables
+3. Config file
+
+#### Flags
+
+Available flags:
+
+```bash
+  -C, --config_file string                       custom config file path
+  -h, --help                                     show help
+  -v, --version                                  show version
+  -d, --dump_config                              dump config to file, specified by config_file (-C) flag
+  -e, --export_headers                           export headers to file
+```
+
+To generate config file with defaults, use the --dump flag, or:
+
+```bash
+go run ./cmd/main.go -d
+```
+
+The default config file path is **project root**, and the default file name is **config.yaml**. This can be overridden by -C flag.
```

Comment:
I'm not sure if `project root` is a precise phrase. I think it's rather `program root`. I mean - the config file should be in the same directory which the compiled executable file is placed. Or I'm wrong - but I recall, I had a problem with that lately.

Reply:
Changed